### PR TITLE
Added elasticsearch.template.properties

### PR DESCRIPTION
**Pull request by araithel**

### CODE REVIEW

File: src/main/resources/elasticsearch.template.properties [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# picturesafe-search requires a configuration file named 'elasticsearch.properties' in the classpath,
+# in which the following default parameters can be overwritten.
+
+
+# --------------------------- Client configuration ----------------------------
+
+## List of host addresses to connect to (blank separated, format <hostname>:<port> or <ip>:<port>)
+#elasticsearch.hosts=localhost:9200
+
+## automatically discover nodes from a running Elasticsearch cluster and set them to an existing RestClient instance
```
Comment (jfrommann) [inline src/main/resources/elasticsearch.template.properties]: Set? Did you mean add?

File: src/main/resources/elasticsearch.template.properties [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# ============ elasticsearch configuration for picturesafe-search =============
+#
+# picturesafe-search requires a configuration file named 'elasticsearch.properties' in the classpath,
+# in which the following default parameters can be overwritten.
```
Comment (jfrommann) [inline src/main/resources/elasticsearch.template.properties]: Sound a bit strange. What about "You can use the following default parameters by leaving them commented or, if you want to change settings, uncomment lines and change the values."?

File: src/main/resources/elasticsearch.template.properties [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## Index name prefix (property 'elasticsearch.index.alias' will be used when empty)
+#elasticsearch.index.name_prefix=
+
+## Index name date format ('yyyyMMdd-HHmmss-SSS' will be used when empty)
+#elasticsearch.index.name_date_format=yyyyMMdd-HHmmss-SSS
+
+#elasticsearch.index.number_of_shards=1
+
+#elasticsearch.index.number_of_replicas=0
```
Comment (jfrommann) [inline src/main/resources/elasticsearch.template.properties]: No comment?

File: src/main/resources/elasticsearch.template.properties [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+##  Maximum number of fields in the index (empty = use elasticsearch default)
```
Comment (jfrommann) [inline src/main/resources/elasticsearch.template.properties]: Better: "... (uses elasticsearch default setting when empty)"

File: src/main/resources/elasticsearch.template.properties [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+#elasticsearch.index.fields_limit=1000
+
+## Maximum result window size (maximum number of hits in a search result)
```
Comment (jfrommann) [inline src/main/resources/elasticsearch.template.properties]: "... (uses elasticsearch default setting when empty)"

File: src/main/resources/elasticsearch.template.properties [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+#elasticsearch.index.max_result_window=10000
+
+# --------------------------- Query configuration -----------------------------
```
Comment (jfrommann) [inline src/main/resources/elasticsearch.template.properties]: "Time zone of the machine picturesafe-search is running on."

File: src/main/resources/elasticsearch.template.properties (hunk 2 above)
Comment (araithel) [reply src/main/resources/elasticsearch.template.properties]: Hm, this is a template file inside the lib, which may change with a release.
I would at least leave the first sentence and then insert yours. Ok?

File: src/main/resources/elasticsearch.template.properties (hunk 1 above)
Comment (araithel) [reply src/main/resources/elasticsearch.template.properties]: It's from the Elastic documentation, but I can change it...

File: src/main/resources/elasticsearch.template.properties (hunk 3 above)
Comment (araithel) [reply src/main/resources/elasticsearch.template.properties]: Speaks for itself, I think. Maybe refer to the Elastic documentation?

File: src/main/resources/elasticsearch.template.properties (hunk 4 above)
Comment (araithel) [reply src/main/resources/elasticsearch.template.properties]: ok

File: src/main/resources/elasticsearch.template.properties (hunk 6 above)
Comment (araithel) [reply src/main/resources/elasticsearch.template.properties]: ok

**Pull request merged**